packages/expo-linear-gradient/src/NativeLinearGradient.android.tsx

```diff
@@ -1,52 +1,51 @@
 import { requireNativeViewManager } from '@unimodules/core';
-import React from 'react';
+import * as React from 'react';
 import { StyleSheet, View } from 'react-native';
 
-type Props = {
-  colors: number[];
-  locations?: number[] | null;
-  startPoint?: Point | null;
-  endPoint?: Point | null;
-} & React.ComponentProps<typeof View>;
+import { NativeLinearGradientProps } from './NativeLinearGradient.types';
 
-type Point = [number, number];
+const NativeLinearGradient: React.FC<NativeLinearGradientProps> = ({
+  colors,
+  locations,
+  startPoint,
+  endPoint,
+  children,
+  style,
+  ...props
+}: NativeLinearGradientProps) => {
+  // TODO: revisit whether we need to inherit the container's borderRadius since this issue has
+  // been resolved: https://github.com/facebook/react-native/issues/3198
+  const flatStyle = StyleSheet.flatten(style) || {};
+  const borderRadius = flatStyle.borderRadius || 0;
```

Comment:
```suggestion
  const borderRadius = flatStyle.borderRadius ?? 0;
```

etc... throughout this file where it makes sense